src/test/java/com/karankumar/bookproject/backend/utils/PredefinedShelfUtilsTest.java

```diff
@@ -121,4 +140,51 @@ void shouldGetAllBooksInChosenShelves() {
         Assertions.assertTrue(actualBooks.containsAll(expectedBooks));
     }
 
-}
+    @Test
+    void testAllPredefinedShelves() {
+        SoftAssertions softly = new SoftAssertions();
+
+        PREDEFINED_SHELVES.stream().forEach(shelfName -> {
+            String errorMsg = MessageFormat.format(ERROR_MSG, shelfName);
+            softly.assertThat(PredefinedShelfUtils.isPredefinedShelf(shelfName)).as(errorMsg).isTrue();
+        });
+
+        softly.assertAll();
+    }
+
+    @Test
+    void testAllPredefinedShelvesLowerCase() {
+        SoftAssertions softly = new SoftAssertions();
+
+        PREDEFINED_SHELVES.stream().map(String::toLowerCase).forEach(shelfName -> {
+            String errorMsg = MessageFormat.format(ERROR_MSG, shelfName);
+            softly.assertThat(PredefinedShelfUtils.isPredefinedShelf(shelfName)).as(errorMsg).isTrue();
+        });
+
+        softly.assertAll();
+    }
+
+    @Test
+    void testAllPredefinedShelvesUpperCase() {
+        SoftAssertions softly = new SoftAssertions();
+
+        PREDEFINED_SHELVES.stream().map(String::toUpperCase).forEach(shelfName -> {
+            String errorMsg = MessageFormat.format(ERROR_MSG, shelfName);
+            softly.assertThat(PredefinedShelfUtils.isPredefinedShelf(shelfName)).as(errorMsg).isTrue();
+        });
+
+        softly.assertAll();
+    }
+
+    @Test
+    void testInvalidShelves() {
```

Comment:
Could you please rename this method such that the name itself reflects what is being tested?